spring-cloud-starter-tencent-polaris-discovery/src/main/java/com/tencent/cloud/polaris/registry/PolarisServiceRegistry.java

```diff
@@ -84,6 +84,7 @@ public void register(Registration registration) {
 			return;
 		}
 		// Register instance.
+		System.out.println("事件注册");
```

Comment:
Use Logger and use English.

Reply:
Yeah, just for proposal, the code not ready yet.